src/custom/components/SearchModal/CurrencyList/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { CurrencyAmount } from '@uniswap/sdk'
+
+import { LONG_PRECISION } from 'custom/constants'
```

Comment:
can just be `constants/index` - i believe that's the way we have it standardised - avoiding adding `custom` in the path

Reply:
https://github.com/gnosis/gp-swap-ui/pull/386